recipes/pyscrap/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pyscrap" %}
+{% set version = "0.0.9" %}
+{% set sha256 = "b15d33ea688381ffccb281bb2a620f13db3a0cb1407544345dca4b80de4e8273" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - import.patch  # [py3k]
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - simplejson
+    - lxml
+    - libxml2
```

Comment:
Are `libxml2` and `libxslt` needed? I couldn't find them referenced anywhere in the source. They seem like they're just dependencies of `lxml`, which will be pulled in automatically.